src/apis/weather-api.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+import { Context } from "../contex.js";
+
+const key = "KEY";
+const instance = axios.create({
+  baseURL: "https://api.openweathermap.org/",
+  timeout: 2000,
+});
```

Comment:
It would be better to leave a blank line here after creating the axios instance.